app/views/layouts/_flash_notices.html.erb

```diff
@@ -13,7 +13,11 @@
         There is a problem
       </h2>
       <div class="govuk-error-summary__body">
-        <p><%= msg %></p>
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <a href="#6-digit-error-summary"><%= msg.html_safe %></a>
```

Comment:
The linter complains about this line because 'html_safe' is considered a dangerous operation here because you don't quite know what the value of 'msg' is and if it has been validated properly. To fix, append 'html_safe' to  https://github.com/alphagov/govwifi-admin/blob/a512d2b1722dd853f81c56a96b3e0de9d3530e77/app/controllers/users/two_factor_authentication_setup_controller.rb#L20

The '<a>' tag shouldn't be here I think. This layout is shared between all views and so we can't put something specific to the 6 digit error summary in here. I know the design system says we should have a link here, but the previous code didn't and to change it would mean changing `Users::TwoFactorAuthenticationSetupController' and views and introducing a form model so we can use `form#govuk_error_summary' instead of a flash message which would probably be better but a larger change that may be outside the scope of this PR.

Reply:
I have appended html_safe to flash alert message. I may have resolved this conversation before the change was pushed but the change is present on the latest commit